keel-veto/src/main/kotlin/com/netflix/spinnaker/keel/veto/VetoController.kt

```diff
@@ -66,6 +66,15 @@ class VetoController(
     return veto.currentRejections()
   }
 
+  @GetMapping(
```

Comment:
a new endpoint to get the resource ids that are currently being vetoed for each application.